docs/source/reference/storage.rst

```diff
@@ -156,6 +156,21 @@ and storage mounting:
         name: romil-output-bucket
         mode: MOUNT
 
+      # *** Uploading multiple files to the same Storage object ***
+      #
+      # The source field in a storage object can also be a list of local paths.
+      # This is useful when multiple files or directories need to be uploaded to the
+      # same bucket.
+      #
+      # Note: If the source list contains a directory, the entire directory is copied
```

Comment:
I think we can make this more concise.
(reword)
```
The basenames of each path in the source list are copied recursively to the root of the bucket.
```

Reply:
great point! while this is concise, I feel it becomes harder to read for a non-systems/os audience. I've kept the original and added this before it.